ui/src/translations/fr.json

```diff
@@ -123,7 +124,7 @@
   "create_new_environment": "Créer un nouvel environnement",
   "description": "Description",
   "add_solution_to_environment": "Ajout d'une solution à l'environnement {environment}",
-  "version_env": "Versions",
+  "version_env": "Version",
```

Comment:
Why `version_env` instead of `version`?

Reply:
because we have another translation  `"version": "MetalK8s Version",`

Reply:
Then `"version": "MetalK8s Version"` should probably be renamed `"metalk8s_version": "MetalK8s Version"` instead